new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java

```diff
@@ -3,37 +3,44 @@
 
 package oracle.weblogic.kubernetes.assertions.impl;
 
+import java.io.IOException;
 import java.util.concurrent.Callable;
 
 import io.kubernetes.client.openapi.ApiException;
-import io.kubernetes.client.openapi.apis.ApiextensionsV1Api;
+import io.kubernetes.client.openapi.Configuration;
 import io.kubernetes.client.openapi.apis.ApiextensionsV1beta1Api;
-import io.kubernetes.client.openapi.apis.ApisApi;
 import io.kubernetes.client.openapi.apis.CustomObjectsApi;
 import io.kubernetes.client.openapi.models.V1beta1CustomResourceDefinition;
+import io.kubernetes.client.util.ClientBuilder;
 
 import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class Domain {
 
-  private static CustomObjectsApi customObjectsApi = new CustomObjectsApi();
-  private static ApiextensionsV1Api apiextensionsV1Api = new ApiextensionsV1Api();
-  private static ApiextensionsV1beta1Api apiextensionsV1beta1Api = new ApiextensionsV1beta1Api();
-  private static ApisApi apisApi = new ApisApi();
+  static {
+    try {
+      Configuration.setDefaultApiClient(ClientBuilder.defaultClient());
+    } catch (IOException ioex) {
+      throw new ExceptionInInitializerError(ioex);
+    }
+  }
 
+  private static final CustomObjectsApi customObjectsApi = new CustomObjectsApi();
+  private static final ApiextensionsV1beta1Api apiextensionsV1beta1Api = new ApiextensionsV1beta1Api();
 
   /**
    * Check if the Domain CRD exists
+   *
    * @return true if domains.weblogic.oracle CRD exists otherwise false
    * @throws ApiException when Domain CRD doesn't exist
    */
   public static boolean doesCRDExist() throws ApiException {
     try {
-      V1beta1CustomResourceDefinition domainBetaCrd =
-          apiextensionsV1beta1Api.readCustomResourceDefinition(
-              "domains.weblogic.oracle", null, null, null);
+      V1beta1CustomResourceDefinition domainBetaCrd
+          = apiextensionsV1beta1Api.readCustomResourceDefinition(
+          "domains.weblogic.oracle", null, null, null);
       assertNotNull(domainBetaCrd);
```

Comment:
please make sure the assertions have failure messages

Reply:
Shall I wait to merge?

Reply:
Fixed it. Please go ahead and merge it.